algorithms/a3c/main.py

```diff
@@ -10,28 +11,41 @@
 """
 
 from __future__ import print_function
-
 import argparse
 import os
+os.environ["OMP_NUM_THREADS"] = "1"  # fixes multiprocessing error on some systems
+import uuid
+import glob
+import json
+import datetime
 
 import torch
 import torch.multiprocessing as mp
+from tensorboardX import SummaryWriter
 
 from gym_ai2thor.envs.ai2thor_env import AI2ThorEnv
```

Comment:
The order should be alphabetic. Algorithms first and them gym

Reply:
I think ordering imports by alphabetic is going too far :P

Reply:
Disagree with you, but fine.